index.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html lang="em">
+    <head>
+        <meta charset="UTF-8">
+        <title>Portfolio</title>
+        <link rel="stylesheet" href="style.css">
+    </head>
+    <body>
+        <div class="container">
+            <div class="box box1">Usama Masood</div>
+            <div class="box box2"><img src="Enabled.png" alt=""></div>
+            
+        </div>
+        <div class="container1">
+            <div class="box box3">Hey There.<br> I'm Usama</div>
+            <div class="box box4">I'am a Software Developer</div>
+            
+        </div>
+        <div class="container2">
+            <div class="container3">
+                <div class="box box5">I can help you build a product , feature or<br>website Look through some of my work<br>and experience! If you like what you see<br>and have a project you need coded, don’t<br>hestiate to contact me</div>
+                <div class="box box6"><img src="Social Media.png" alt=""></div>
+            </div>
+            <div class="box box7"><img src="Illustration.png" alt=""></div>
+        </div>
```

Comment:
- [ ] Kindly wrap this section in a `section` tag. It represents a generic standalone section of the document, so use it. [Check this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section) for more info. The container of the section is a section tag.

 - [ ] The main title should be wrapped with an `h1` tag. If you want that google (and other search engines) know about a specific topic, in this case, you, use an `h1` tag. [Check this link](https://www.reliablesoft.net/h1-tag/) for more info. Kindly fix that.

 - [ ] The subtitle uses a header tag `hx (examples: <h1>,<h2>,<h3>, etc...)`. This way will be semantically correct and the screen readers and other technologies will thank you. Kindly wrap the subtitle following the `h` tag hierarchy and double-check for typos. 👍 
 
- [ ] The paragraph should be wrapped with a `p` tag. This way will be semantically correct and the screen readers and other technologies will thank you. Kindly change that.

- [ ] The list of icons uses a `ul` tag to group them, and `li` per icon; each icon has a link (`a` tag). Apart from being semantically correct, the `a` tag will let you connect your social networks with the icons, using `href`. Kindly remove that image and you could copy those icons from the Figma template and add them as individual images in the `a` tag inside the list so you can add custom links for your social pages to them.

- [ ]  You added the background image as an HTML image element. If an image is not part of the content (aesthetic reasons) use CSS background-image. [Check this link](https://stackoverflow.com/questions/492809/when-to-use-img-vs-css-background-image) for more info. Kindly remove the image and use the CSS background-image property to add it.